styles/style.css

```diff
@@ -0,0 +1,75 @@
+.container {
+   
+    width: 1900px;
+    height: 900px;
+    display: grid; 
+    grid-template-columns: 1fr 1fr ;
+    grid-template-rows: 1fr 1fr 1fr ;
+    
+}
+
+.flex-container {
+   
+    width: 1900x;
+    height: 900px;
+    display: flex; 
+    flex-wrap:nowrap;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    
+}
+ h1{
+
+    background-color: pink;
+    grid-column: span 5;
+    grid-row: 1;
```

Comment:
Not all of your h1 tags are in grids. Try to make your rules more specific. And for the one that _is_ in a grid, there are only 2 columns, but your span indicates to span 5.